packages/legend-studio/src/stores/EditorGraphState.ts

```diff
@@ -392,6 +392,32 @@ export class EditorGraphState {
     return entityChanges;
   }
 
+  *loadEntityChangesToGraph(changes: EntityChange[]): GeneratorFn<void> {
+    try {
+      assertTrue(
+        this.editorStore.isInFormMode,
+        'Applying changes only supported in form mode',
+      );
+      const entities =
+        this.editorStore.graphManagerState.graph.allOwnElements.map((element) =>
+          this.editorStore.graphManagerState.graphManager.elementToEntity(
+            element,
+          ),
+        );
+      const updatedEntities =
+        this.editorStore.localChangesState.patchChangeState.applyEntityChanges(
+          [...entities],
```

Comment:
Don't think we need `[...entities]`. I even think we can inline everything in the `updateGraphAndApplication(...)` call :)